apis/r/src/Makevars.in

```diff
@@ -1,27 +1,24 @@
 CXX_STD = CXX17
 
 ## We need the TileDB Headers, and for macOS aka Darwin need to set minimum version 10.14 for macOS
-#PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ -DR_BUILD @cxx17_macos@ -Iinclude
-PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ @cxx17_macos@ -Iinclude
+PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ @cxx17_macos@
```

Comment:
did you mean to remove `-Iinclude` at the end?

Reply:
I did. That is a leftover from when we had copied files around into src.

Now, the universe being what it is I of course got myself an immediate CI fail that revealed the `pkg-config` was not actually asking for `tiledb` _and_ `tiledbsoma`.  I fixed that, and it may help.  The breaking build ended up with an empty `-I` which then destroyed the subsequent argument (for the present and needed headers from Rcpp).  

Worst case I think I can stick the voodoo -Iinclude back in.